packages/message-backend/src/email/create/templates/partials/paymentRow.hbs

```diff
@@ -52,36 +53,45 @@
                                                 {{#if value}}
                                                     <tr>
                                                         <td width="100%" class="purchase_item">
-                                                            <span class="f-fallback no-wrap pb-1" style="font-size:14px;">
+                                                            <span class="f-fallback no-wrap pb-1" style="font-size:16px;">
                                                                 {{value}}
                                                             </span>
                                                         </td>
                                                     </tr>
                                                 {{/if}}
-                                                {{#if @last}}
```

Comment:
Why this is removed, this is needed because email providers does not support css like normally, this is needed for outlook example.